configuration.json

```diff
@@ -0,0 +1,30 @@
+{
+  "blank_image": {
+    "type": "RGBA",
```

Comment:
I wouldn't consider hex codes and fonts etc to be considered UI strings since they're not seen by the user.

The strings that I would consider to be "UI strings" would be:
"No image found" (Done)

These ones in menu.py:
"Open File(s)"
"Open Folder"
"Refresh"
"Exit"
"Dark"
"Light"

since these are all seen by the user. In the future, those are the strings that would need to be translated.

I do apologise, I should've outlined this better in the issue! 

Mind, what you've done here certainly isn't bad at all, and I think there could be some value in having the sizes and fonts abstracted out like you've done here! So here's what I'd like (please!):
- The strings in menu.py I outlined above to be put in here
- Colours are being handled in this other PR as part of the saving themes work (https://github.com/cBridges851/tkinter-image-viewer/pull/26), so I think they can be removed from here. 
- Fonts and sizes (as in where we have 500 and 300) to be under `default_display_settings.json`, or something to that effect.
- If en-GB.json and default_display_settings.json could be in folders (e.g. `lang` and `settings` or `configuration`), that would be great :)

Hope that all sounds ok?